httpx_oauth/clients/twitch.py

```diff
@@ -0,0 +1,93 @@
+from typing import Any, Dict, List, Optional, Tuple, cast
+
+from httpx_oauth.errors import GetIdEmailError
+from httpx_oauth.oauth2 import BaseOAuth2
+
+AUTHORIZE_ENDPOINT = "https://id.twitch.tv/oauth2/authorize"
+ACCESS_TOKEN_ENDPOINT = "https://id.twitch.tv/oauth2/token"
+REVOKE_TOKEN_ENDPOINT = "https://id.twitch.tv/oauth2/revoke"
+BASE_SCOPES = [
+    "user:read:email",
+    "user:read:follows",
```

Comment:
Does all those scopes are required in a basic use case (i.e. get the id and email)? If not, I prefer to keep this list as short as possible, and let the end-user add its own set of scopes to fit their needs.